src/encoded/schemas/changelogs/library.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for library.json
 
+### Schema version 9
+
+* *fragmentation_method* instead of being a single value, it is now an array of values. 
```

Comment:
Are we sure we don't want it to be called `fragmentation_methods` now that it is an array?

Reply:
makes sense